fastparquet/compression.py

```diff
@@ -5,11 +5,32 @@
 
 # TODO: enable ability to pass kwargs to compressor
 
-compressions = {'GZIP': gzip.compress,
-                'UNCOMPRESSED': lambda x: x}
-
-decompressions = {'GZIP': gzip.decompress,
-                  'UNCOMPRESSED': lambda x: x}
+compressions = {
+    'UNCOMPRESSED': lambda x: x
+}
+decompressions = {
+    'UNCOMPRESSED': lambda x: x
+}
+try:
+    compressions['GZIP'] = gzip.compress
+    decompressions['GZIP'] = gzip.decompress
+except AttributeError:
+    # Python2.7
+    def gzip_compress(data, compresslevel=9):
+        from io import BytesIO
+        bio = BytesIO()
+        f = gzip.GzipFile(mode='wb',
+                          compresslevel=compresslevel,
```

Comment:
Suggest that the compression level could be a module constant, and also applied to gzip.compress for py3.